### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by j-f1**
This isn’t used for our Docker builds but it is a good starting point that makes it fairly easy to clone this repo + get started running the latest stable image. I personally would recommend this over running via raw Docker or trying to run locally.

### CODE REVIEW

**Review by joaociocca — APPROVED** [review]
Works like a charm! Would be even better if it already included, even if as comments, the other available environment options, like `ACTUAL_HTTPS_KEY` and `ACTUAL_HTTPS_CERT`.

**Comment by j-f1** [comment]
Added a bunch more comments!

**Review by joaociocca — APPROVED** [review]
Whoa, this was even better than I expected. Awesome <3

File: docker-compose.yml [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+      # This line makes Actual available at port 5006 of the device you run the server on,
+      # i.e. http://localhost:5006. You can change the first number to change the port, if you want.
       - "5006:5006"
+    environment:
+      # Uncomment any of the lines below to set configuration options.
+      # - ACTUAL_HTTPS_KEY=/data/selfhost.key
+      # - ACTUAL_HTTPS_CERT=/data/selfhost.crt
+      # - ACTUAL_NORDIGEN_SECRET_ID=xxxx
+      # - ACTUAL_NORDIGEN_SECRET_KEY=xxxx
+      # See all optoins and more details at https://actualbudget.github.io/docs/Installing/Configuration
```
Comment (Flightkick) [inline docker-compose.yml:15]: Typo `optoins` > `options`

Comment (j-f1) [reply docker-compose.yml]: Thanks!

**Comment by j-f1** [comment]
Merging since this is essentially a docs-only update.

**Pull request merged**